### PR TITLE
Fix pytest exception asserts

### PR DESCRIPTION
**Pull request by mehanig**
Work still in progress. Python3: all tests passed, Python2: 2 tests failed.

### CODE REVIEW

**Comment by mehanig** [comment]
2 failed, 56 passed in 3.47 seconds.
2 tests failing: one for Cassandra and one for tests/test_oai_harvester.py: but there is deeper problem, probably with Normalize function.

**Comment by fabianvf** [comment]
As we talked about in the issue, if you want to scrap python 2 compatibility, feel free to.

**Comment by mehanig** [comment]
@fabianvf perfect!

**Comment by fabianvf** [comment]
looks like tests failed due to the ```class X(.. , metaclass=..)``` syntax

**Comment by mehanig** [comment]
Sorry, my bad

File: tests/test_base.py [hunk 1]
```diff
@@ -33,8 +33,7 @@ def test_beat_schedule(self, mock_registry):
             assert(isinstance(val.run_at, dict))
 
     def test_beat_schedule_adds(self, mock_registry):
-        class TestClass(object):
-            __metaclass__ = HarvesterMeta
+        class TestClass(object, metaclass=HarvesterMeta):
```
Comment (fabianvf) [inline tests/test_base.py]: still have the bad class declaration here.

**Pull request merged**